README.md

```diff
@@ -74,13 +74,10 @@ module.exports = {
       withoutCLIReporting: false,
     },
     castWallet: {
-      enabled: false,
       // Optional parameters below
```

Comment:
Maybe they are not optional anymore?

Reply:
They are still optional; only one of them is required. If you use an account, you cannot use a keystore, and if you use a keystore, you cannot use an account. The password file can be absent